src/proper-parenthetics/proper_parenthetics.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import linked_list
+
+
+def proper_parenthetics(a_string):
+    """Return broken(-1), balanced(0) or open(1) parenthesis for string."""
+    sll = linked_list.LinkedList()
+    a_string = a_string[::-1]
```

Comment:
Does using the LinkedList here actually win you any simplicity that using the original string itself would not?  In other words, can you do exactly what you've done here just with the string?

Can you think of another of our data structures from this last week that *would*?